tests/integration/cairo_compiler/tests_compilation_test.py

```diff
@@ -0,0 +1,130 @@
+from pathlib import Path
+import pytest
+
+import cairo_python_bindings
+
+from tests.data.cairo1_contracts import (
+    CAIRO_BINDINGS_CONTRACT_ENUM,
+    CAIRO_BINDINGS_TESTS,
+)
+from tests.integration.conftest import CreateProtostarProjectFixture
+from tests.integration._conftest import ProtostarFixture
+
+
+@pytest.fixture(name="protostar")
+def protostar_fixture(create_protostar_project: CreateProtostarProjectFixture):
+    with create_protostar_project() as protostar:
+        protostar.build_sync()
+        yield protostar
+
+
+@pytest.fixture(name="prepared_files")
+def prepared_files_fixture(protostar: ProtostarFixture):
+    cairo_example_path = Path("./src/example.cairo")
+    cairo_tests_path = Path("./src/tests.cairo")
+    sierra_path = Path("./src/example.sierra")
+    casm_path = Path("./src/example.casm")
+
+    paths = {
+        "cairo_example": cairo_example_path,
+        "cairo_tests": cairo_tests_path,
+        "sierra": sierra_path,
+        "casm": casm_path,
+    }
+
+    for _, path in paths.items():
+        protostar.create_files({str(path): ""})
+        assert path.exists()
+
+    with open(cairo_example_path, "w") as file:
+        file.write(CAIRO_BINDINGS_CONTRACT_ENUM)
+
+    with open(cairo_tests_path, "w") as file:
+        file.write(CAIRO_BINDINGS_TESTS)
```

Comment:
This thing is redundant. If you pass a `Path` object to the function, it'll automatically read the file, and if you pass a `str`, it will get the code from the string.

Reply:
Sorry, but I don't understand :/ What function do you mean?

Reply:
`ProtostarFixture::create_files`/`ProtostarFixture::create_contracts`

Reply:
[extract fixture to a sep file](https://github.com/software-mansion/protostar/pull/1316/commits/25d34a76296e38402292118940d46c8917b2c850)